docker-compose.test.yml

```diff
@@ -2,17 +2,20 @@ version: '2.3'
 
 services:
   # Override the supervisor service configuration. The tests will
-  # be run through the `sut` service
+  # be run through the `sut` service and the running superviisor will be
+  # tested through the `balena-supervisor-sut` service
   balena-supervisor:
+    command: sleep infinity
```

Comment:
We don't need to override the entrypoint for this service. As long as nothing requires it, it will not be started.

This is because Flowzone runs `docker compose up sut --exit-code-from sut` and not `docker compose up --exit-code-from sut`.

https://github.com/product-os/flowzone/blob/854998ed291f416958a6eeb9b6a21306eb5bbd66/flowzone.yml#L2037

This is the same command run by DockerHub so we are following their process.

Reply:
Fixed @klutchell

Reply:
Looks good, I think we can squash the commits though, since mine didn't fix anything.

Reply:
Other than that LGTM